### PR TITLE
Sch 177544 a11y

### PR DESCRIPTION
**Pull request by shewood**
## [ADO-177544](https://dev.azure.com/VP-BD/DECD/_workitems/edit/177544)

[AB#177544](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/177544)

### Description of proposed changes:
Fix A11Y issues for multiple pages as indicated in the tasks outline in the ADO story number above.

### CODE REVIEW

File: components/contact/ContactProvinceRow.tsx [hunk 1]
```diff
@@ -33,27 +33,27 @@ export const ContactProvinceRow = ({
...
               <div
-                className="col-span-2 md:col-span-1 py-3 select-text cursor-default"
+                className="col-span-2 cursor-default select-text py-3 md:col-span-1"
                 key={i}
               >
-                <strong className="prose prose-strong:text-xl prose-strong:font-display prose-p:text-xl prose-p:font-display">
-                  <Markdown>{`${ap(rowItem.content, ' ')}`}</Markdown>
-                </strong>
-
+                {' '}
```
Comment (krischarbonneau) [inline components/contact/ContactProvinceRow.tsx:44]: Accidental space?

Comment (shewood) [reply components/contact/ContactProvinceRow.tsx:44]: Hmm. good question. I will look

Comment (shewood) [reply components/contact/ContactProvinceRow.tsx]: Removed the space.

**Review by krischarbonneau — APPROVED** [review]
LGTM!

**Pull request merged**